activities/Abecedarium.activity/lib/tutorial.js

```diff
@@ -1,286 +1,238 @@
 					title: l10n.get("TutoPlay1Title"),
-					content: l10n.get("TutoPlay1Content")
+					intro: l10n.get("TutoPlay1Content"),
 				},
 				{
 					element: "#play_playTypeButton2",
-					placement: "bottom",
+					position: "bottom",
 					title: l10n.get("TutoPlay2Title"),
-					content: l10n.get("TutoPlay2Content")
+					intro: l10n.get("TutoPlay2Content"),
 				},
 				{
 					element: "#play_playTypeButton3",
-					placement: "top",
+					position: "bottom",
 					title: l10n.get("TutoPlay3Title"),
-					content: l10n.get("TutoPlay3Content")
+					intro: l10n.get("TutoPlay3Content"),
 				},
 				{
 					element: "#play_playTypeButton4",
-					placement: "top",
+					position: "top",
 					title: l10n.get("TutoPlay4Title"),
-					content: l10n.get("TutoPlay4Content")
+					intro: l10n.get("TutoPlay4Content"),
 				},
 				{
 					element: "#play_playTypeButton5",
-					placement: "top",
+					position: "top",
 					title: l10n.get("TutoPlay5Title"),
-					content: l10n.get("TutoPlay5Content")
+					intro: l10n.get("TutoPlay5Content"),
 				},
 				{
 					element: "#play_playTypeButton6",
-					placement: "top",
+					position: "top",
 					title: l10n.get("TutoPlay6Title"),
-					content: l10n.get("TutoPlay6Content")
+					intro: l10n.get("TutoPlay6Content"),
 				},
 				{
 					element: ".entryPlayFrom",
-					placement: "bottom",
+					position: "bottom",
 					title: l10n.get("TutoPlayEntryBoxTitle"),
-					content: l10n.get("TutoPlayEntryBoxContent")
+					intro: l10n.get("TutoPlayEntryBoxContent"),
 				},
 				{
 					element: "#play_check",
-					placement: "left",
+					position: "left",
 					title: l10n.get("TutoCheckTitle"),
-					content: l10n.get("TutoCheckContent")
+					intro: l10n.get("TutoCheckContent"),
 				},
 				{
 					element: "#play_caseButton",
-					placement: "bottom",
+					position: "bottom",
 					title: l10n.get("TutoFontTitle"),
-					content: l10n.get("TutoFontContent")
+					intro: l10n.get("TutoFontContent"),
 				},
 				{
 					element: "#play_languageButton",
-					placement: "left",
+					position: "left",
 					title: l10n.get("TutoLangTitle"),
-					content: l10n.get("TutoLangContent")
+					intro: l10n.get("TutoLangContent"),
 				},
 				{
 					element: "#play_home_home",
-					placement: "bottom",
+					position: "bottom",
 					title: l10n.get("TutoHomeTitle"),
-					content: l10n.get("TutoHomeContent")
+					intro: l10n.get("TutoHomeContent"),
 				},
 				{
 					element: "#play_back",
-					placement: "bottom",
+					position: "bottom",
 					title: l10n.get("TutoBackTitle"),
-					content: l10n.get("TutoBackContent")
+					intro: l10n.get("TutoBackContent"),
 				},
 				{
 					element: "#play_filter",
-					placement: "bottom",
+					position: "bottom",
 					title: l10n.get("TutoLearnBoxThemeTitle"),
-					content: l10n.get("TutoLearnBoxThemeContent")
-				}
-			]);
+					intro: l10n.get("TutoLearnBoxThemeContent"),
+				},
+			);
 		}
-		tour.init();
-		tour.start(true);
+		debugger;
```

Comment:
This line should be removed.

Reply:
oops! i forgot to review my git diff!

Reply:
Yeah i did that because of introJs showing weird behaviors when **scrollbar** isn't there..!
![image](https://user-images.githubusercontent.com/90092555/202352387-07d86a6d-0d2a-4652-bd56-9a06e24218b6.png)

I'll try to fix it and update you

Reply:
> This line should be removed.

oops! i forgot to review my git diff!